features/roles.feature

```diff
@@ -20,33 +20,44 @@ Feature: Manage WordPress roles
     When I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 0/1 roles.
+      No changes necessary for 'author' role.
+      Success: Roles reset 0/1.
       """
 
     When I run `wp cap remove author read`
     And I run `wp role reset author`
     Then STDOUT should be:
       """
-      Success: Reset 1/1 roles.
+      Restored 10 capabilities to and removed 9 capabilities from 'author' role.
```

Comment:
We are removing entire role and adding it back but we can get the diff of capability and show message like in this case `Restored 1 capability and removed 0 capabilities from 'author' `

Do you want to show message in case of 0 capability changed ?
Like in above case message should it only display `Restored 1 capability from 'author'`?